docs/cloud/rahti/usage/projects_and_quota.md

```diff
@@ -37,20 +36,20 @@ For more information about using the web interface, refer to the
 out which version of the documentation to look at in the web interface by
 clicking the question mark symbol in the top bar and selecting "About".
 
-### Extra information about CSC computing projects in Rahti
+### Extra information about CSC computing projects in Rahti 1
 
 Projects in OpenShift are separate from CSC computing projects. A single CSC
 computing project can have access to multiple projects in OpenShift.
-Each CSC computing project with access to Rahti receives a *group* in
+Each CSC computing project with access to Rahti 1 receives a *group* in
 OpenShift.
 
 !!! note
-    Rahti can be used free of charge for open research and
+    Rahti 1 can be used free of charge for open research and
     education in Finnish universities and polytechnics.
 
 Any usage within OpenShifts projects are billed
 to the billing unit quota of the CSC project selected. Note that the selected CSC project must
-have Rahti access and the user creating a project must be a member of that computing project,
+have Rahti 1 access and the user creating a project must be a member of that computing project,
 otherwise the OpenShift project creation will fail.
 
 * If you would like to know which CSC computing projects you are a member of, you
```

Comment:
default billing projects are not supported in Rahti-2

Reply:
fixed